src/SFA.DAS.EmployerCommitmentsV2.Web/Models/Cohort/WithTransferSenderRequest.cs

```diff
@@ -0,0 +1,11 @@
+﻿using SFA.DAS.Authorization.ModelBinding;
+
+namespace SFA.DAS.EmployerCommitmentsV2.Web.Models.Cohort
+{
+    public class WithTransferSenderRequest : IAuthorizationContextModel
+    {
+        public string AccountHashedId { get; set; }
+        public long AccountId { get; set; }
+
+    }
+}
```

Comment:
Spoke to Chris and this might not be problem because of the Mappers, but because they should all return different view models this may not be a problem

Reply:
Actually, could we call it CohortsByAccountRequest

Reply:
done.